jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaLayerConfigurations.java

```diff
@@ -134,10 +146,13 @@ public static Builder builder() {
   }
 
   private final ImmutableMap<LayerType, LayerConfiguration> layerConfigurationMap;
+  private final ImmutableMap<LayerType, String> defaultExtractionPathMap;
 
   private JavaLayerConfigurations(
-      ImmutableMap<LayerType, LayerConfiguration> layerConfigurationsMap) {
-    this.layerConfigurationMap = layerConfigurationsMap;
+      ImmutableMap<LayerType, LayerConfiguration> layerConfigurationsMap,
+      ImmutableMap<LayerType, String> defaultExtractionPathMap) {
+    layerConfigurationMap = layerConfigurationsMap;
```

Comment:
Perhaps we should name these both `layerConfigurationsMap` so this can be the more familiar `this.x = x` form?